src/sentry/static/sentry/app/views/settings/project/projectTeams.jsx

```diff
@@ -126,11 +65,12 @@ class ProjectTeams extends AsyncView {
     });
   };
 
-  handleAdd = selection => {
-    if (this.state.loading) return;
-
-    const team = this.state.allTeams.find(tm => tm.id === selection.value);
+  handleAdd = teamSlug => {
+    if (this.state.loading) {
```

Comment:
imo we should enforce `{}` in linter

Reply:
I would love that. I'll look into enabling those rules.